### PR TITLE
feat: support property-level examples

### PR DESCRIPTION
**Pull request by barrett-schonefeld**
Related issue: #467 

Updates:
- Process property-level examples.
- Merge property-level examples with schema-level examples provided with `example` keyword.

Tests:
- Update tests that use schemas with property-level examples to reflect additional generated examples.
- Add tests for each function to generate static params from property-level examples.

### CODE REVIEW

File: test/specs/openapi/test_examples.py [hunk 1]
```diff
@@ -9,50 +11,110 @@
...
+    }
+
+
+@pytest.fixture(scope="module")
+def schema_with_examples(dict_with_examples) -> BaseOpenAPISchema:
+    return schemathesis.from_dict(dict_with_examples)
+
+
+@pytest.fixture(scope="module")
+def schema_with_property_examples(dict_with_property_examples) -> Dict[str, Any]:
```
Comment (Stranger6667) [inline test/specs/openapi/test_examples.py:116]: The return type should be `BaseOpenAPISchema`

File: test/specs/openapi/test_examples.py [hunk 2]
```diff
@@ -84,7 +152,7 @@ def test_get_request_body_examples(endpoint):
 
 
 def test_get_static_parameters_from_examples(endpoint):
-    static_parameters_list = examples.get_static_parameters_from_examples(endpoint, "examples")
+    static_parameters_list = examples.get_static_params_from_examples(endpoint, "examples")
```
Comment (Stranger6667) [inline test/specs/openapi/test_examples.py:155]: Could you, please, rename this function back to `get_static_parameters_from_examples`? I'd prefer slightly more verbose naming

File: src/schemathesis/specs/openapi/examples.py [hunk 3]
```diff
@@ -37,22 +79,31 @@ def get_static_parameters_from_example(endpoint: Endpoint) -> Dict[str, Any]:
...
+def get_static_params_from_properties(endpoint: Endpoint) -> Dict[str, Any]:
+    endpoint_def = endpoint.definition.resolved
+    return {**get_param_example_from_properties(endpoint_def), **get_request_body_example_from_properties(endpoint_def)}
+
+
 def get_strategies_from_examples(endpoint: Endpoint, examples_field: str = "examples") -> List[SearchStrategy[Case]]:
     strategies = [
-        get_strategy(endpoint, static_parameters)
-        for static_parameters in get_static_parameters_from_examples(endpoint, examples_field)
+        get_strategy(endpoint, static_params)
```
Comment (Stranger6667) [inline src/schemathesis/specs/openapi/examples.py:97]: Could you, please, rename `static_params` to `static_parameters` here as well?

File: README.rst [hunk 4]
```diff
@@ -344,8 +344,7 @@ Or add this decorator to your test if you use Schemathesis in your Python tests:
     def test_api(case):
         ...
 
-**NOTE**. Schemathesis does not support examples in individual properties that are specified inside Schema Object.
-But examples in `Parameter Object <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#parameter-object>`_, `Media Type Object <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject>`_ and `Schema Object <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#schema-object>`_ are supported.
+**NOTE**. Schemathesis now supports examples in individual properties.
```
Comment (Stranger6667) [inline README.rst:347]: Hm, probably we can remove “now”, since Readme context doesn’t imply a history of changes

File: docs/changelog.rst [hunk 5]
```diff
@@ -6,6 +6,11 @@ Changelog
 `Unreleased`_
 -------------
 
+Added
+~~~~~
+
+- support for property-level examples. `#637`_
```
Comment (Stranger6667) [inline docs/changelog.rst:12]: Should it be 467 as added below?

Comment (barrett-schonefeld) [reply docs/changelog.rst:12]: Ah, I couldn't remember if it was issue number or PR number.

File: src/schemathesis/specs/openapi/examples.py [hunk 6]
```diff
@@ -38,21 +83,33 @@ def get_static_parameters_from_example(endpoint: Endpoint) -> Dict[str, Any]:
...
     ]
-    static_parameters = get_static_parameters_from_example(endpoint)
-    if static_parameters:
-        strategy = get_strategy(endpoint, get_static_parameters_from_example(endpoint))
-        strategies.append(strategy)
+    for static_parameters in static_parameters_union(
+        get_static_parameters_from_example(endpoint), get_static_parameters_from_properties(endpoint)
+    ):
+        strategy = get_strategy(endpoint, static_parameters)
+        if strategy is not None:
```
Comment (Stranger6667) [inline src/schemathesis/specs/openapi/examples.py:111]: Btw, after recent refactoring, `get_strategy` always returns `SearchStrategy`, so this condition can be removed

**Pull request merged**